test/fixtures/usagePlan/index.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports.handler = (event, context, callback) => {
```

Comment:
Instead of creating new fixture, let's simply extend existing one (e.g. `apiGateway` or `apiGatewayExtended`)

See e.g. how in ALB tests we extend `function` fixture to test everything: https://github.com/serverless/serverless/blob/80e70e7affd54418361c4d54bdef1561af6b8826/lib/plugins/aws/package/compile/events/alb/lib/healthCheck.test.js#L121